.github/workflows/primer_run_main.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     timeout-minutes: 45
     strategy:
       matrix:
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.8", "3.12-dev"]
```

Comment:
If we're going to run the primer on 3.12dev, we should also change it in the PR job and comment job. I'd rather keep the primer more stable by not doing it and just waiting for 3.12 final, although it could be interesting to do on a fork as an experiment.